app/process-image.js

```diff
@@ -109,38 +121,60 @@ class ImageProcessing {
 				new_image = image.scaleToFit(1440, 2560, Jimp.RESIZE_HERMITE);
 
 				// determine if image is a raid image or not
-				let pixel_check = Jimp.intToRGBA(image.getPixelColor(30, 300));
```

Comment:
@Googer can you please add this pixel check back in?  it was meant for pictures of things like rocket map, screenshots of a gym from the gym list on the profile page of the game.

Reply:
I'm a bit torn on that since in testing there was a legitimate screenshot that by chance got filtered out by that.  Probably a fluke I know but hey 😛

Reply:
Oh yeah? can you share the screenshot somewhere? i'm curious how that happened.  A single pixel seemed extremely unlikely to do that in pokemon go since all pictures are pretty colorful.

Reply:
I'll see if I can dig it up since I have a big load of them and didn't mark it specifically when I'd hit it.

Reply:
ok this is the only outstanding issue left.  I would like to see the screenshot that failed, from my perspective removing this check corrects 1 screenshot while adding additional work to every other screenshot.  It feels better to make other screenshots check more quickly, while very rarely failing on others.  Maybe we can alter this code a bit to "fix" the 1 screenshot, rather than removing the check all together?

Reply:
That optimization saves a trivial amount of time that's not worth worrying about - even without it, a non-raid (which is the worst-case since it never gets short-circuited and goes through the full iteration of both pixel-check loops) is determined in <1ms on my laptop, let alone on a 'real' server.  In comparison, the image scaling performed beforehand takes anywhere from ~1/2 s to ~3 s (example is on a non-raid image):

```
[2017-12-16 21:05:41:749 DEBUG ImageProcessor] Scaling image
[2017-12-16 21:05:44:777 DEBUG ImageProcessor] Scaled
[2017-12-16 21:05:44:777 DEBUG ImageProcessor] Determining if raid
[2017-12-16 21:05:44:777 DEBUG ImageProcessor] Determined
[2017-12-16 21:05:44:777 DEBUG ImageProcessor] null
```

Reply:
So you're saying.... it does save time though? ;) so lets add it back in, even if it's trivial, when dealing with potentially dozens of screenshots being analyzed simultaneously, on a slower machine (say a free AWS EC2 instance with a dual core), this pixel check can save a bit more processing power.

Not everyone has a beefed up system, we should keep everyone's setup in mind.  There's no reason not to have the code other than 1 screenshot supposedly didn't work, but I have yet to see any proof of that.  If you have the screenshot, again please present it, and lets talk about how we can tweak the idea to work better for both worlds.

Reply:
You're trading correctness for an imperceptible time savings.  My laptop is definitely slower than even the free AWS tier, and as it shows, the resize code that's performed even on a non-raid screenshot is on the order of 3 magnitudes slower than the full looping check.  It's not worth saving <1ms (exact amount not known because that's the granularity of the logging) when there's already a *much* larger time sink ahead of it in the processing pipeline.